testsuite/features/init_clients/sle_ssh_minion.feature

```diff
@@ -22,6 +22,7 @@ Feature: Bootstrap a Salt host managed via salt-ssh
 # Package 'sle-manager-tools-release' is automatically installed during bootstrap and
 # stays installed after removal of channel containing it. So it is not possible to update it.
 # Package needs to be removed from highstate to avoid failure when updating it.
+@skip_sp_migration
```

Comment:
What about naming it the same that the other tag, but adding "skip_".
So : "skip_service_pack_migration"
Or calling both: (skip_)sp_migration

Reply:
I have changed it to: `(skip_)service_pack_migration`